_posts/2019-08-23-hobby-apps.md

```diff
@@ -5,7 +5,7 @@ subtitle: What Kind to Make?
 
 ---
 
-In the past couple years, I have spoken to several aspiring iOS developers about what kind of hobby app they should make after escaping the [tutorial trap](https://twitter.com/seanallen_dev/status/1097166143884677120). By "hobby app", I mean an app for which one does not intend to be paid at all by an employer or enough by users to cover one’s living expenses. As someone who used hobby apps to jump-start his iOS-development career and who continues to develop hobby apps, I am interested in this question. I share my thinking in this post with the hope of providing [thought-food](https://www.merriam-webster.com/dictionary/neologism) to anyone considering development of a hobby app.
+In the past couple of years, I have spoken to several aspiring iOS developers about what kind of hobby app they should make after escaping the [tutorial trap](https://twitter.com/seanallen_dev/status/1097166143884677120). By "hobby app", I mean an app for which one does not intend to be paid at all by an employer or enough by users to cover one’s living expenses. As someone who used hobby apps to jump-start his iOS-development career and who continues to develop hobby apps, I am interested in this question. I share my thinking in this post with the hope of providing [thought-food](https://www.merriam-webster.com/dictionary/neologism) to anyone considering development of a hobby app.
```

Comment:
Although "couple years" is not incorrect, "couple of years" reads a bit better, so I made the change.